packages/web3-eth/src/web3_eth.ts

```diff
@@ -259,6 +818,108 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 		return rpcMethodsWrappers.getTransactionCount(this, address, blockNumber, returnFormat);
 	}
 
+	/**
+	 * @param transaction The {@link Transaction} or {@link TransactionWithLocalWalletIndex} to send.
+	 * @param returnFormat ({@link DataFormat} defaults to {@link DEFAULT_RETURN_FORMAT}) Specifies how the return data should be formatted.
+	 * @param options A configuration object used to change the behavior of the `sendTransaction` method.
```

Comment:
we should also add an example for usage of `options?: SendTransactionOptions,` options A configuration object used to change the behavior of the `sendTransaction` method.

Reply:
I [added](https://github.com/ChainSafe/web3.js/pull/5251/commits/04881110d73286f137462c136397d76d3e39661a) an exmaple, but I think a better description of the properties of `SendTransactionOptions` should be done in the type's docs